### ISSUE TITLE
Debugging to_executorch() call

### ISSUE THREAD

**Issue by stanislawmorawski-digica** [pytorch/executorch#1890]
I was converting ultralytics/yolov5 from torch hub to executorch. Unfortunately one of the layers was unsupported, and the program exited with an error on this line: `executorch_program: exir.ExecutorchProgramManager = edge_program.to_executorch()`. The traceback was:

![image](https://github.com/pytorch/executorch/assets/104894679/8ccbd60f-d0e0-4346-b991-64f165148908)

I've looked into the file being executed and this is what I saw:

```python
class BaseModel(nn.Module):

    def _forward_once(self, x, p, v):
        for m in self.model: # nn.Sequential
            ...
            x = m(x)  # run
            ...
        return x
```

What is the recommended way to find out which layer in the `nn.Sequential` failed?

Given that:

* Logging & printing is disabled
* Debugging with breakpoints doesn't work

I've ended up monkey-patching the model and executing the nn.Sequential by hand:

```python
class DebugModel(nn.Module):

    def forward(self, x):
        x0 = self.model[0](x)
        x1 = self.model[1](x0)
        x2 = self.model[2](x1)
        x3 = self.model[3](x2)
        x4 = self.model[4](x3)
        x5 = self.model[5](x4)
        x6 = self.model[6](x5)
        x7 = self.model[7](x6)
        x8 = self.model[8](x7)
        x9 = self.model[9](x8)
        x10 = self.model[10](x9)
        x11 = self.model[11](x10)
        x12 = self.model[12]([x11, x6])  # [-1, 6],
        x13 = self.model[13](x12)
        x14 = self.model[14](x13)
        x15 = self.model[15](x14)
        x16 = self.model[16]([x15, x4])  # [-1, 4],
        x17 = self.model[17](x16)
        x18 = self.model[18](x17)
        x19 = self.model[19]([x18, x14])  # [-1, 14],
        x20 = self.model[20](x19)
        x21 = self.model[21](x20)
        x22 = self.model[22]([x21, x10])  # [-1, 10],
        x23 = self.model[23](x22)
        x24 = self.model[24]([x17, x20, x23])  # [17, 20, 23]

        return x24
```

Executing this way I've found out that 11th layer is actually nn.Upsample which is unsupported.  But my question is: Is there a better way? Is there a `torch.export.log()` or something similar?

**Comment by iseeyuan** [comment]
@Olivia-liu , is it possible to print out the module stack when a node is not supported? cc @tarun292 @Jack-Khuu

**Comment by hietalajulius** [comment]
@stanislawmorawski-digica were you able to resolve the issue / do you have an example how you are creating the export?

**Comment by liuyibox** [comment]
Hi @hietalajulius @stanislawmorawski-digica , are you able to successfully export any yolo models to executorch? I have been stuck on this point for a while.

**Comment by hietalajulius** [comment]
Hi @liuyibox! I have not tried, I'm assuming it does not work until someone posts a working example :D

**Comment by Olivia-liu** [comment]
@iseeyuan What do you mean by module stack? Like the internal implementation details of nn.Sequential? I don't think we preserve that even at torch.export(), no?

**Comment by iseeyuan** [comment]
@Olivia-liu Yes it may be hard to print which layer the error is from in nn.Sequential. The issue happened a while ago. Not sure if it has been improved. @angelayi Do you know if there are any fixes on this?

**Comment by nil-is-all** [comment]
Hi @angelayi, any update on this?
